src/selector-generation/produce-selector/options-converter.js

```diff
@@ -24,19 +24,19 @@ export const configToOptions = selectorConfig => ({
 /**
  * Decides if ids should be used in selector generation
  * @param {SelectorConfig} selectorConfig
- * @param {string} a
- * @param {string} idName
+ * @param {string} attributeName this is not used by us in this case is alwatys equal with 'id' string from "[id='idName']"
```

Comment:
There is only a typo here in the comment, and a similar one in the ignoreClassFn. `alwayts`